graphql/util.go

```diff
@@ -618,17 +618,9 @@ func ModifyVersion(version model.Version, user user.DBUser, proj *model.ProjectR
 			}
 		}
 	case SetPriority:
-		var projId string
-		if proj == nil {
-			projId, err := model.GetIdForProject(version.Identifier)
-			if err != nil {
-				return http.StatusNotFound, errors.Errorf("error getting project ref: %s", err)
-			}
-			if projId == "" {
-				return http.StatusNotFound, errors.Errorf("project for %s came back nil: %s", version.Branch, err)
-			}
-		} else {
-			projId = proj.Id
+		projId := version.Identifier
+		if projId == "" {
+			return http.StatusNotFound, errors.Errorf("Could not find project for version %s", version.Id)
```

Comment:
Would this ever happen? 🤔 i.e. would `version.Identifier` ever be `""`

Reply:
Someone on app can verify but i don't think thats possible. But if it is its a good defensive programming technique to case on it. Otherwise i think it will just return false on line 632

Reply:
checked in the prod database and there does seem to be some without an identifier but they seem to be weird without a project so i think it's safe to assume that it shouldnt be empty